.ci/azure/pypi.yml

```diff
@@ -0,0 +1,35 @@
+jobs:
+  # Push to TestPyPI (only after push to main)
+  - job: TestPyPI
+    # condition: startsWith(variables['build.sourceBranch'], 'refs/heads/main')
+    steps:
+      # Checkout simpeg repo, including tags.
+      # Sync tags and disable shallow depth to get the SimPEG version.
+      - checkout: self
+        fetchDepth: 0
+        fetchTags: true
+        displayName: "Checkout repository (including tags)"
+
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+        displayName: "Setup Python 3.9"
+
+      - script: |
+          pip install setuptools setuptools_scm wheel twine
+        displayName: "Install build dependencies"
+
+      - script: |
+          python setup.py sdist
+        displayName: "Create source distribution for simpeg"
+
+      - script: |
+          twine upload --repository testpypi dist/*
+        displayName: "Upload to Test PyPI"
+        env:
+          TWINE_USERNAME: $(test.twine.username)
```

Comment:
```suggestion
          TWINE_USERNAME: $(twine.username)
```

Reply:
Or just write out `__token__`

Reply:
Ohh, only for the username. Got it.

Reply:
The username is literally just `__token__` (it’s not even a secret variable on azure)

Reply:
I see. Thanks for the tip.